lib/structures/Message.js

```diff
@@ -220,12 +220,18 @@ class Message extends Base {
                 data.content = "This server has failed Discovery activity requirements for 3 weeks in a row. If this server fails for 1 more week, it will be removed from Discovery.";
                 break;
             }
+            case MessageTypes.THREAD_CREATED: {
+                break;
```

Comment:
```suggestion
                data.content = `${this.author.mention} started a thread: ${data.content}. See all threads.`;
                break;
```

Reply:
@abalabahaha leaving this up to you as I prefer not replacing the message content with a fixed localised string for a message that has no content in the first place